tests/determineEventSource.test.js

```diff
@@ -0,0 +1,40 @@
+import test from 'tape';
```

Comment:
@acastro this file is a duplicate?  if not, it should be in same directory as target file being tested.

Reply:
moves to aws tests